src/components/Layout/Header.tsx

```diff
@@ -18,6 +36,7 @@ export default function Header() {
           <SearchInput />
         </section>
       </header>
+      {isOpen && <OffCanvas setOpen={setOpen} />}
```

Comment:
Solved it by moving the `OffCanvas` component outside of the header and putting only the button there instead.

Reply:
Okay great. Good luck with your project. I will be around 😊